scripts/aggregate_result.py

```diff
@@ -106,7 +109,7 @@ def aggregate_results(model: str) -> Dict[str, float]:
 
```

Comment:
```suggestion
import os # <-適当なことろにimportする

c_result_dir_is_file = os.path.isfile(result_dir)
c_result_dir_exists = os.path.isdir(result_dir)
assert not c_result_dir_is_file, f"outpur dir {c_result_dir_is_file} is a file"
if not c_result_dir_exists:
  os.makedirs(c_result_dir_exists)
```

result_dirが存在しない・result_dirがファイルの場合への対応